bears/general/CLOCBear.py

```diff
@@ -0,0 +1,65 @@
+import re
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='cloc')
+class CLOCBear:
+    """
+    Summarises a file or number of files in directory structure
+    with total lines, number of comment lines, number of actual code lines
+    using CLI tool cloc
+    """
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {DistributionRequirement('cloc')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Number of comment lines',
```

Comment:
I cannot see any useful aspects that can be used here, but `Commented Code`. And that too, doesn't describe enough. May be we should add one aspect called `Statistics`.
But then also, it doesn't make sense to use it under CAN_DETECT. Its rather calculating, than just detecting. So, I guess we should also add one more `CAN_xxx` thing, like `CAN_FIND` or `CAN_COMPUTE`.
Lemme know how should go I ahead.
Irrespective, I'm just pushing changes as requested from other comments.